src/components/ccsds_command_depacketizer/gen/templates/assembly/name_ccsds_cosmos_commands.txt

```diff
@@ -0,0 +1,34 @@
+{% set plugin_format_dictionary = {'F': 'FLOAT', 'U': 'UINT', 'E': 'UINT', 'I': 'INT'} %}
+{% for id, command in commands.items() %}
+Command {{ name }} {{ command.suite.component.instance_name }}-{{ command.name }} Big_Endian "{{ command.description }}"
+{# The ccsds command header #}
+{% set header_bit_position = [] %}
+{% for field_name, field in ccsds_primary_header_model.fields.items() %}
+  Parameter {{ field.name }} {{ field.start_bit }} {{ field.size -}}
+{% if loop.last %}{% set _ = header_bit_position.append(field.start_bit + field.size) %}{% endif %} {{ plugin_format_dictionary[field.format.type[0]] }} 0 {{ 2**field.size - 1 }} {% if field.default_value != number %}0{% else %}{{field.default_value}}{% endif %} "{{ field.description|replace('\n','') }}"
+{% if field.is_enum %}
+{% for literals in field.datatype.model.literals %}
+    State {{ literals.name }} {{ literals.value }}
+{% endfor %}{% endif -%}{% endfor %}
+{% for field_name, field in ccsds_command_secondary_header.fields.items() %}
+  Parameter {{ field.name }} {{ header_bit_position|last + field.start_bit }} {{ field.size }} {{ plugin_format_dictionary[field.format.type[0]] }} 0 {{ 2**field.size - 1 }} {% if field.default_value != number %}0{% else %}{{field.default_value}}{% endif %} "{{ field.description|replace('\n','') }}"
+{% if field.is_enum %}
+{% for literals in field.datatype.model.literals %}
+    State {{ literals.name }} {{ literals.value }}
+{% endfor %}{% endif -%}{% endfor %}
+  ID_Parameter Adamant_Command_Id 64 16 UINT MIN MAX {{ command.id }} "Adamant command ID"
+{# The command argument definitions. #}
+{% if command.type %}{% if command.type_model %}{% for field in command.type_model.flatten() %}
+{# if field.format.length, "specific_name+field.name" #}
+{#% for attr in field.datatype.__dict__.keys() %}
```

Comment:
We need to remove all commented out code before merging.

Reply:
Addressed in [5a27090](https://github.com/lasp/adamant/pull/21/commits/5a27090bbf5e9cfe4236af4fb6254bae9ed5795e).